driver-sync/src/test/functional/com/mongodb/client/ClientSideEncryptionBsonSizeLimitsSpecification.groovy

```diff
@@ -39,10 +39,10 @@ import static util.JsonPoweredTestHelper.getTestDocument
 
 class ClientSideEncryptionBsonSizeLimitsSpecification extends FunctionalSpecification {
 
-    private final String collectionName = 'ClientSideEncryptionBsonSizeLimitsSpecification'
+    private final String collName = 'ClientSideEncryptionBsonSizeLimitsSpecification'
```

Comment:
Side effect of Spock dependency update? Theres a `getCollectionName` method in the `FunctionalSpecification` and spock complained they clashed.